docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc

```diff
@@ -4,85 +4,91 @@
 
 == Visual Editor
 
-Prisma Cloud supports the capability of a Visual Editor where you can create custom build policies for  your templates using the existing fields.
-The Visual Editor is a suitable option when you want to create custom policies that include Attribute checks with a support of AND/OR logic.
-To help you create a custom policy using the Visual Editor, you will see an example of custom build policy for S3 Bucket ACL where log delivery is not recommended.
-
+The 'Visual Editor' supports custom build policies for *IaC*, *Secrets* and *Licenses*, using Attribute checks that support of AND/OR logic.
```

Comment:
> using Attribute checks that support of AND/OR logic

I'm trying to think of how to not complicate things, but IaC supports AND/OR, Secrets and License policies only supports OR logic.

Reply:
I will come up with something